src/content/blog/2024/office_hours.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Introducing nf-core office hours'
+subtitle: "A new initative from nf-core! Here's what you can expect."
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Introducing nf-core office hours'
+    start: 2024-01-12T16:14:00+01:00
+    end: 2024-01-13T16:14:00+01:00
+pubDate: 2024-01-12T16:14:00+01:00
+authors:
+    - 'christopher-hakkaart'
+label:
+    - 'community'
+---
+
+# nf-core office hours
+
+Slack serves as an outstanding platform for asynchronous collaboration.
+Nonetheless, there are moments when engaging in live discussions about your code proves beneficial.
+We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.
+
+# What you can expect
+
```

Comment:
```suggestion

nf-core office hours will be held online at regular times, open to anyone to drop in and out as they'd like.
It can be nice to just sit with some other people working on similar stuff (even virtually).
And of course it's a great opportunity to chat with others about Nextflow and nf-core development.

```